gm4_calling_bell/data/gm4_calling_bell/functions/load.mcfunction

```diff
@@ -0,0 +1,7 @@
+execute if score gm4 load.status matches 1 if score gm4_trades load.status matches 1 run scoreboard players set gm4_calling_bell load.status 1
+execute unless score gm4 load.status matches 1 run data modify storage gm4_log queue append value {type:"missing",module:"Calling Bell",require:"Gamemode 4"}
+execute unless score gm4_trades load.status matches 1 run data modify storage gm4:log queue append value {type:"missing",module:"Calling Bell",require:"lib_trades"}
```

Comment:
you'll need to verify a minor load score to verify the new function tag you added exists